xml-js/xml-js.d.ts

```diff
@@ -0,0 +1,95 @@
+// Type definitions for xml-js 0.9.5
+// Project: https://github.com/nashwaan/xml-js
+// Definitions by: Denis Carriere <https://github.com/DenisCarriere>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "xml-js" {
+    interface Attribute {
+        [key: string]: string
+    }
+
+    type Types = "comment" | "element" | "cdata" | "text";
+
+    export interface ElementCompact {
+        [key: string]: any
+        _attributes?: Attribute
+        _cdata?: string
+        _comment?: string
+        _declaration?: DeclarationCompact
+        _text?: string
+    }
+
+    export interface Element {
+        attributes?: Attribute
+        cdata?: string
+        comment?: string
+        declaration?: Declaration
+        elements?: Array<Element>
+        text?: string
+        type?: Types
+        name?: string
+    }
+
+    interface DeclarationCompact {
+        _attributes?: {
+            version?: string
+            encoding?: string
+        }
+    }
+
+    interface Declaration {
+        attributes?: {
+            version: string
+            encoding: string
+        }
+    }
+
+    interface XML2JSOptions extends ChangingKeyNames, IgnoreOptions {
+        compact?: boolean
+        spaces?: number
+        trim?: boolean
+        sanitize?: boolean
+        nativeType?: boolean
+        addParent?: boolean
+        alwaysChildren?: boolean
+    }
+
+    interface JS2XMLOptions extends ChangingKeyNames, IgnoreOptions {
+        spaces?: number
+        compact?: boolean
+        fullTagEmptyElement?: boolean
+    }
+
+    interface IgnoreOptions {
+        ignoreDeclaration?: boolean
+        ignoreAttributes?: boolean
+        ignoreComment?: boolean
+        ignoreCdata?: boolean
+        ignoreText?: boolean
+    }
+
+    interface ChangingKeyNames {
+        declarationKey?: string
+        attributesKey?: string
+        textKey?: string
+        cdataKey?: string
+        commentKey?: string
+        parentKey?: string
+        typeKey?: string
+        nameKey?: string
+        elementsKey?: string
+    }
+
+    type True = true;
+
+    interface JS2XMLOptionsCompact extends JS2XMLOptions {
+        compact: True;
```

Comment:
Could I get some advice on how to solve this?
Adding `compact=true` changes the input, I'm using Typescript 2.0 and it works for me but fails on the build.
Thanks in advance